s3/filestorage.go

```diff
@@ -410,8 +410,8 @@ func (s *SimpleStorageService) UploadArtifact(
 			uploadParams,
 			func(req *request.Request) {
 				// ExpireTime will presign URI to expire after
-				// 5 minutes
-				req.ExpireTime = time.Minute * 5
+				// 60 minutes
+				req.ExpireTime = 60 * time.Minute
```

Comment:
how about making it configureable?

Reply:
Nice idea, let me rework it to make it configurable (both upload & download).